lib/components/Select2/Select.tsx

```diff
@@ -19,6 +22,33 @@ import { CustomSelectProps } from "./Select.types";
 
 const defaultStatus = "info";
 
+const { ValueContainer } = components;
+
+interface CustomValueContainerProps extends ValueContainerProps {
+  getValue: () => Options<any>;
```

Comment:
isn't this function already available from the interface ValueContainerProps? this custom prop interface can probably be removed.

Reply:
Yes, you're right, I removed the CustomValueContainerProps. But I couldn't make the changes you mentioned above work unfortunatelyf